pm_uep_opt_write/run.sh

```diff
@@ -0,0 +1,11 @@
+for x in $1/*/*/*.yaml 
```

Comment:
I reckon that now pm-uep-opt can return a non-zero code, you can reduce this to the one-liner you had before and just add an extra bit somewhere to propagate the exit code out, e.g. https://itectec.com/unixlinux/bash-how-to-get-this-script-to-error-exit-based-on-result-of-for-loop/. I don't think we need extra logging output since pm-uep-opt already prints all the useful information.

Reply:
replaced with error propagation from for loop. This took a while to get right 😅